examples/workflow-glsp/src/direct-task-editing/direct-task-editor.ts

```diff
@@ -20,28 +20,60 @@ import {
     EditorContextService,
     getAbsoluteClientBounds,
     GLSPActionDispatcher,
+    hasStringProp,
     ILogger,
-    isSetContextActionsAction,
-    isSetEditValidationResultAction,
     LabeledAction,
+    Operation,
     RequestContextActions,
     RequestEditValidationAction,
-    SModelElement,
+    SetContextActions,
+    SetEditValidationResultAction,
     SModelRoot,
     toActionArray,
     ValidationDecorator,
     ValidationStatus,
     ViewerOptions
 } from '@eclipse-glsp/client';
 import { inject, injectable } from 'inversify';
-import { SModelIndex, TYPES } from 'sprotty/lib';
+import { ModelIndexImpl, TYPES } from 'sprotty/lib';
 import { DOMHelper } from 'sprotty/lib/base/views/dom-helper';
 import { isTaskNode, TaskNode } from '../model';
 
-export class ApplyTaskEditOperation implements Action {
-    static readonly KIND = 'applyTaskEdit';
-    readonly kind = ApplyTaskEditOperation.KIND;
-    constructor(readonly taskId: string, readonly expression: string) {}
+/**
+ * Is send from the {@link TaskEditor} to the GLSP server
+ * to execute a task edit operation.
+ */
+export interface ApplyTaskEditOperation extends Operation {
+    /**
+     * The unique action kind
+     */
```

Comment:
I'd spare such comments, also if it hurts to be "incomplete". But imho they just add noise and add no information.